etna/records/widgets.py

```diff
@@ -30,7 +30,7 @@ def get_instance(self, pk):
         """Fetch related instance on edit form."""
         try:
             return Record.search.get(iaid=pk)
-        except (KongAPIError, SearchManagerException):
```

Comment:
- APIModel.search raises DeprecationWarning, so Record.search could raise 
- API for Search does not have iaid param, instead Fetch has iaid param

Reply:
Thanks Allan. Good spot. I've replace the call to `Record.search`.